### DISCUSSION TITLE
Prefixing REST routes in v5

### DISCUSSION THREAD

**Discussion by nammuMax** [feathersjs/feathers#3331] [Q&A] [answered]
Hi,

Developping an app with FeathersJS (v5 and Koa) and Svelte for the frontend, I want to compile my frontend in static and put it in the "public" directory to be served from the Feathers app.

Is there a way to prefix all my Feathers services routes with "/api" to not get conflicts with the frontend routes ?

I tried to check with "koa-router" but couldn't make it work. Looked into creating a subapp but this will put everything with the prefix (I don't mint websockets routes being not prefixed as it won't create conflicts with the frontend), and I'm thinking about manually prefixing all my services paths with "/api" and changing my calls in the frontend to calls services with prefixes, but yeah, was wondering if there was a better way to do it ?

Would be better if I can still cal `feathers.service('my-service')...` from the frontend, and just have the REST routes prefixed with /api.

Thanks

**Comment by noor-tg** [comment, marked as answer]
I think it is better to use something like nginx as frontend web server. And use different port for the api.

**Reply by ericuldall** [reply]
or deploy it for free on netlify :D https://www.netlify.com/with/svelte/

Either way I agree with @nour-dev ! Decoupling your client from your api is the best way to go. Theres no need to redeploy your api when only changing client code and vice versa.
